packages/webpack-bundler-runtime/src/initContainerEntry.ts

```diff
@@ -26,7 +26,11 @@ export function initContainerEntry(
     ...remoteEntryInitOptions,
   });
 
-  federationInstance.initShareScopeMap(name, shareScope);
+  federationInstance.initShareScopeMap(
+    name,
+    shareScope,
+    remoteEntryInitOptions?.shareScopeMap,
```

Comment:
Here, the share scope map is only used as one of the values, otherwise the subsequent expansion value can only be continuously increased